src/A1_csv_to_json.py

```diff
@@ -390,7 +393,9 @@ def create_json_from_csv(input_directory, filename, parameters):
             single_dict.update({column: column_dict})
             # add exception for energyStorage
             if filename == "energyStorage":
-                storage_dict = add_storage(column, input_directory)
+                storage_dict = add_storage(
```

Comment:
```suggestion
                # Add storage component parameters, whose input file is named after the names of storages in of the energyStorage.csv
                storage_dict = add_storage_components(
```

Reply:
As unchanged, I can not suggest changes to the function name "add_storage" where it is defined in the code. Ie, find that lines and change to "add_storage_components", as this makes clear that energyStorages.csv is not opened but instead the sub-file that defines the three parts in/out power and storage capacity.

Reply:
I changed the function name to add_storage_components()

Reply:
changed it!